src/Integration.Vsix/Analysis/IssueConsumerFactory_IssueHandler.cs

```diff
@@ -89,7 +89,10 @@ private void MarkSuppressedIssues(IEnumerable<IAnalysisIssueVisualization> issue
             {
                 var filterableIssues = issues.OfType<IFilterableIssue>().ToArray();
 
-                var matches = issuesFilter.GetMatches(filterableIssues);
+                var matches = filterableIssues
```

Comment:
Can't we just do the check inside the `foreach` in line 98+ and get rid of the `matches` list?

i.e. `issue.IsSuppressed = suppressedIssueMatcher.SuppressionExists(issue)`

Reply:
Oops, yeah that is simpler :D